src/ts/components/subtitlesettings/characteredgecolorselectbox.ts

```diff
@@ -0,0 +1,57 @@
+import { SubtitleSettingSelectBox, SubtitleSettingSelectBoxConfig } from './subtitlesettingselectbox';
+import { UIInstanceManager } from '../../uimanager';
+import { PlayerAPI } from 'bitmovin-player';
+import { i18n } from '../../localization/i18n';
+
+/**
+ * A select box providing a selection of different character edge colors.
+ *
+ * @category Components
+ */
+export class CharacterEdgeColorSelectBox extends SubtitleSettingSelectBox {
+
+  constructor(config: SubtitleSettingSelectBoxConfig) {
+    super(config);
+
+    this.config = this.mergeConfig(config, {
+      cssClasses: ['ui-subtitlesettingscharacteredgecolorselectbox'],
```

Comment:
I started to use kebab case for CSS classes for new components. I know that this format aims for consistency but it's very hard to read 🫣 Fine for me to keep it like this but I'd be also fine with switching to kebab case.

Reply:
Yes, I was aiming for constistency. But I also agree that this is quite hard to read. Changed in 6c81793374643d435561823974469e2ec03e4c9b